inter/time.go

```diff
@@ -9,6 +9,8 @@ import (
 type (
 	// Timestamp is a UNIX nanoseconds timestamp
 	Timestamp uint64
+	// Duration is a UNIX nanoseconds duration
+	Duration uint64
```

Comment:
I am not quite sure about this declaration.
- The standard library offers time. Duration, which is an int64, but end up casting this duration to int64 to construct `big.Int` quite often. 
- Additionally, this file seems to declare a specific timestamp in unix format, not a library of time utils.